src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -193,12 +202,13 @@ def __call__(self,
 class Channel(_base_channel.Channel):
     _loop: asyncio.AbstractEventLoop
     _channel: cygrpc.AioChannel
-    _unary_unary_interceptors: Optional[Sequence[UnaryUnaryClientInterceptor]]
+    _unary_unary_interceptors: Sequence[UnaryUnaryClientInterceptor]
```

Comment:
Nit: Looks like we can change this annotation to `List[UnaryUnaryClientInterceptor]` since the member is guaranteed to always be a list.